frontend/src/components/navbars/GlobalNavBar/index.js

```diff
@@ -32,7 +33,7 @@ export default () => {
                 <img
                     src={require('assets/logos/wordmark_black.png')}
                     className={classes.wordmark}
-                    alt="Junction wordmark"
+                    alt={config.PLATFORM_OWNER_NAME + ' wordmark'}
                 />
                 <UserMenu />
```

Comment:
Let's change this to use a url, so we can customise the logo without causing a git diff.